### PR TITLE
Update panasonic_viera.markdown

### PR DESCRIPTION
**Pull request by MatthiasFraass**
added Tx-65Hz1500 because it works without problems.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Added the `TX-65HZ1500` key to the list of known valid keys for Panasonic Viera models in the integration documentation.

### CODE REVIEW

File: source/_integrations/panasonic_viera.markdown [hunk 1]
```diff
@@ -166,5 +166,6 @@ The list with all known valid keys can be found [here](https://github.com/floria
 - TX-P50GT30Y
 - TX-P50GT60E
 - TX-65HZ1000W
+  TX-65HZ1500
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]: Missing list item marker

```suggestion
- TX-65HZ1500
```

**Review by frenck — APPROVED** [review]
Thanks, @MatthiasFraass 👍 

../Frenck

**Comment by MatthiasFraass** [comment]
added Tx-H65Hz1500

**Comment by MatthiasFraass** [comment]
closed

**Comment by frenck** [comment]
@MatthiasFraass Why did you close this PR?

**Comment by MatthiasFraass** [comment]
I'm too stupid to use PR. It is my first one ;).
I reopened it now. What's the next move?

**Comment by frenck** [comment]
> I'm too stupid to use PR. It is my first one ;)

Hehe no worries!

> I reopened it now. What's the next move?

Well, when you've opened a PR, it is wait and see :)

So... wait and see 😄

**Review by frenck — APPROVED** [review]
Thanks, @MatthiasFraass 👍 

../Frenck

**Comment by frenck** [comment]
Done! Congrats on your first pull request 🎉 

../Frenck

**Pull request merged**